openshift-knative-operator/pkg/monitoring/rbac_proxy_injection.go

```diff
@@ -31,18 +33,26 @@ func InjectRbacProxyContainer(deployments sets.Set[string], cfg base.ConfigMapDa
 		Requests: defaultKubeRBACProxyRequests,
 		Limits:   corev1.ResourceList{},
 	}
-	if cfg != nil && cfg["deployment"] != nil {
-		if cpuRequest, ok := cfg["deployment"]["kube-rbac-proxy-cpu-request"]; ok {
-			resources.Requests["cpu"] = resource.MustParse(cpuRequest)
-		}
-		if memRequest, ok := cfg["deployment"]["kube-rbac-proxy-memory-request"]; ok {
-			resources.Requests["memory"] = resource.MustParse(memRequest)
-		}
-		if cpuLimit, ok := cfg["deployment"]["kube-rbac-proxy-cpu-limit"]; ok {
-			resources.Limits["cpu"] = resource.MustParse(cpuLimit)
-		}
-		if memLimit, ok := cfg["deployment"]["kube-rbac-proxy-memory-limit"]; ok {
-			resources.Limits["memory"] = resource.MustParse(memLimit)
+	logLevel := KubeRbacProxyLogLevel
+	if cfg != nil {
+		if cfg["deployment"] != nil {
+			if cpuRequest, ok := cfg["deployment"]["kube-rbac-proxy-cpu-request"]; ok {
+				resources.Requests["cpu"] = resource.MustParse(cpuRequest)
+			}
+			if memRequest, ok := cfg["deployment"]["kube-rbac-proxy-memory-request"]; ok {
+				resources.Requests["memory"] = resource.MustParse(memRequest)
+			}
+			if cpuLimit, ok := cfg["deployment"]["kube-rbac-proxy-cpu-limit"]; ok {
+				resources.Limits["cpu"] = resource.MustParse(cpuLimit)
+			}
+			if memLimit, ok := cfg["deployment"]["kube-rbac-proxy-memory-limit"]; ok {
+				resources.Limits["memory"] = resource.MustParse(memLimit)
+			}
+			if cfg["logging"] != nil {
```

Comment:
This is not under the `deployment` right? Then should the if not be one level to the left?